### DISCUSSION TITLE
Unhead v1.1.28

### DISCUSSION THREAD

**Discussion by LorenzoRottigni** [unjs/unhead#171] [General]
Hi i think that there's a little bug with new themeColor array feature because now by passing a string like this:

```
// to catch the first request in SSR
useServerSeoMeta({
  themeColor: () => theme.value?.hexPrimary,
  msapplicationTileColor: () => theme.value?.hexPrimary,
  ogLocale: () => useLocaleSanitizer(locale.value as CountryCode),
})

useSeoMeta({
  themeColor: () => theme.value?.hexPrimary,
  msapplicationTileColor: () => theme.value?.hexPrimary,
  ogLocale: () => useLocaleSanitizer(locale.value as CountryCode),
})
```

It creates an element like this:
<meta 0="#" 1="1" 2="6" 3="a" 4="3" 5="4" 6="a" name="theme-color">

**Comment by harlan-zw** [comment]
Thanks! Have just pushed up a fix in `1.1.29`

**Comment by LorenzoRottigni** [comment]
Thanks @harlan-zw ! 
For completeness i report you that to make it working i had to define it as:
themeColor: () => ({ content: theme.value?.hexPrimary }),
By using the example described in there https://github.com/unjs/unhead/releases/tag/v1.1.28: 
 [
    { content: 'cyan', media: '(prefers-color-scheme: light)' },
    { content: 'black', media: '(prefers-color-scheme: dark)' },
  ],
I had 2 meta tag theme-color in my HTML page, one reactive and one not. I suggest you to also check this.
I have to wait the new release of nuxt3 in order to integrate this fix right? Should I manually change the version of unhead to 1.1.29 in my package.json alternatively?

Thanks!

**Reply by harlan-zw** [reply]
Hm.. not entirely sure it will work when you have multiple :thinking: 

You should be able to run `nuxi upgrade --force` and it _should_ just work. The main thing it does is delete your lock file / node_modules and re-installs which checks for newest versions.

**Comment by LorenzoRottigni** [comment]
Hi, using v1.1.30 it works as expected, thanks :)
